scripts/index.js

```diff
@@ -1 +1,95 @@
-// add your solutions here
\ No newline at end of file
+// Positive Numbers
+
+function returnsPositiveNumbers(givenArray) {
+    newArray = [];
+    const positiveNumbers = givenArray.filter(num => {
+        return num > 0;
+    })
+    newArray.push(positiveNumbers);
+    return newArray;
+}
+
+// Even Numbers
+
+function returnEvenNumbers(givenArray) {
+    newArray = [];
+    const evenNumbers = givenArray.filter(num => {
+        return num % 2 === 0;
+    })
+    newArray.push(evenNumbers);
```

Comment:
same as above for `filter()` method (no need to push to a new array), but you've got the idea of doing it without using a loop, so good job!